src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -412,15 +412,23 @@ private void InitializeSuffixes()
             AddSuffix(new[] {"SHIPNAME", "NAME"}, new SetSuffix<string>(() => Vessel.vesselName, RenameVessel, "The KSP name for a craft, cannot be empty"));
             AddSuffix("TYPE", new SetSuffix<string>(() => Vessel.vesselType.ToString(), RetypeVessel, "The Ship's KSP type (e.g. rover, base, probe)"));
             AddSuffix("SENSORS", new Suffix<VesselSensors>(() => new VesselSensors(Vessel)));
-            AddSuffix("TERMVELOCITY", new Suffix<double>(() => VesselUtils.GetTerminalVelocity(Vessel)));
+            AddSuffix("TERMVELOCITY", new Suffix<double>(() => { throw new KOSAtmosphereDeprecationException("17.2", "TERMVELOCITY", "<None>", string.Empty);}));
```

Comment:
I am not sure that the concept of `termvelocity` is still useful so i removed it.